src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -215,6 +215,25 @@ public void GetFolderPath_Unix_PersonalIsHomeAndUserProfile()
             Assert.Equal(Environment.GetEnvironmentVariable("HOME"), Environment.GetFolderPath(Environment.SpecialFolder.UserProfile));
         }
 
+        [Theory]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]  // Tests OS-specific environment
+        [InlineData(Environment.SpecialFolder.ApplicationData)]
+        [InlineData(Environment.SpecialFolder.Desktop)]
+        [InlineData(Environment.SpecialFolder.DesktopDirectory)]
+        [InlineData(Environment.SpecialFolder.Fonts)]
+        [InlineData(Environment.SpecialFolder.MyMusic)]
+        [InlineData(Environment.SpecialFolder.MyPictures)]
+        [InlineData(Environment.SpecialFolder.MyVideos)]
+        [InlineData(Environment.SpecialFolder.Templates)]
+        public void GetFolderPath_Unix_SpecialFolderNotCreated_DoesNotThrow(Environment.SpecialFolder folder)
```

Comment:
Since this deletes "important" folders, albeit if they don't prevoiusly exist, you might consider [Outerloop] out of paranoia.

Reply:
How does this test relate to https://github.com/maryamariyan/corefx/blob/32a11618b153175f38d38b512202de11c1e5f712/src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs#L254-L294?  If it's testing "DoNotVerify" (which I take to be the primary purpose given the "SpecialFolderNotCreated_DoesNotThrow" part of the name), do we have overlap with that other test?